src/app/(content-pages)/transform-property/TransformPropertyPage.tsx

```diff
@@ -128,26 +133,32 @@ export default function TransformPropertyPage() {
           />
         </div>
 
-        <h2 className="heading-2xl font-bold mt-8">Advanced Interventions</h2>
-        <p className="body-md">
+        <h2 className="heading-2xl font-bold mt-8 mb-6">
+          Advanced Interventions
+        </h2>
+        <p className="body-md mb-4">
           Different properties have different opportunities and different
           challenges. Some properties might be a great spot for a rain garden
           but not for a playground. Likewise, you wouldn't want to start a
           community garden in a vacant lot that doesn't get any sun. To better
           understand what your options are for a specific property, try filling
-          out the{" "}
+          out the
           <a
+            target="_blank"
+            rel="noopener noreferrer nofollow"
             href="https://detroitfuturecity.com/whatwedo/land-use/DFC-lots/"
-            className="link"
+            className="link mx-1"
           >
             Detroit Future City vacant property quiz
-          </a>{" "}
-          (the recommendations work for Philadelphia too!). It's meant to help
```

Comment:
could we actually leave in the `{" "}` spaces instead of adding a margin? that would actually be the proper way of adding in copy spacing adjacent to components.